tango/integrations/datasets/__init__.py

```diff
@@ -63,6 +69,29 @@ def convert_to_tango_dataset_dict(hf_dataset_dict):
         return DatasetDict(splits=hf_dataset_dict)
 
 
+T = Union[ds.Dataset, ds.DatasetDict]
+
+
+@Format.register("datasets")
+class DatasetsFormat(Format[T]):
+    """
+    This format writes a :class:`datasets.Dataset` or :class:`datasets.DatasetDict` to disk
+    using :meth:`datasets.Dataset.save_to_disk()`.
+
+    It is the default :class:`~tango.format.Format` for the :class:`LoadDataset` step.
+    """
+
+    VERSION = 1
+
+    def write(self, artifact: T, dir: PathOrStr):
+        dataset_path = Path(dir) / "data"
+        artifact.save_to_disk(str(dataset_path))
```

Comment:
I'm assuming this writes multiple files, and that's why you stuck it into a directory?

Reply:
Yes